specification/search/data-plane/Azure.Search/preview/2021-04-30-Preview/searchservice.json

```diff
@@ -8469,6 +8474,16 @@
           "format": "int32",
           "x-nullable": true,
           "description": "If set, the number of parallel calls that can be made to the Web API."
+        },
+        "authResourceId": {
+          "type": "string",
+          "x-nullable": true,
+          "description": "If set, indicates that this skill should use managed identity. This will be used as the resource id for indicating the scope of the authentication token."
```

Comment:
line 8481.  I think this description is too light. I had to go back to our old email threads to understand what this is supposed to be.  What do you think about this:

"Applies to custom skills that connect to external code in an Azure function or some other application that provides the transformations. This value should be the application ID created for the function or app when it was registered with Azure Active Directory. When specified, the custom skill connects to the function or app using a managed ID (either system or user-assigned) of the search service and the access token of the function or app."

ID should either be capitalized or spelled out as "identifier".